README.md

```diff
@@ -19,40 +19,57 @@ cd coreos-vagrant
 
 3) Startup and SSH
 
-There are two "providers" for Vagrant with slightly different instructions.
-Follow one of the following two options:
+There are three "providers" for Vagrant with slightly different instructions.
+Follow one of the following three options:
 
-**VirtualBox Provider**
+- **VirtualBox Provider**
 
-The VirtualBox provider is the default Vagrant provider. Use this if you are unsure.
+  The VirtualBox provider is the default Vagrant provider. Use this if you are unsure.
 
-```
-vagrant up
-vagrant ssh
-```
+  ```
+  vagrant up
+  vagrant ssh
+  ```
 
-**VMware Provider**
+ - **VMware Provider**
```

Comment:
This is getting bulleted under the Virtualbox Provider - it shouldn't be, right?